superset-frontend/src/explore/components/PropertiesModal/index.tsx

```diff
@@ -82,26 +81,6 @@ export default function PropertiesModal({
             label: `${owner.first_name} ${owner.last_name}`,
           })),
         );
-
-        if (chart.query_context === null) {
```

Comment:
@hughhhh I had to move this to `ExploreChartPanel.jsx` because the screenshot calls the standalone version of the chart, and the screenshot request was not hitting this part of the code.